src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -901,10 +904,8 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
 
             // if primary key have multiple columns, we must recreate it without dropped column
             if (clonedTable.primaryColumns.length > 0) {
-                const pkName = this.connection.namingStrategy.primaryKeyName(clonedTable.name, clonedTable.primaryColumns.map(column => column.name));
```

Comment:
can you please explain this change?

Reply:
Unfortunately I can't.
I have no idea why I've changed it like that and when I look at it right now it just feels wrong. This code changes primary key and has nothing to do with enum type.

I remember working on it(took me a while to realize why `primaryColumns.length` is compared to 0), but I no longer think it's a valid change. I've checked changes after making PR, so at the time I thought change was necessary. I'll remove them since I have no clue why it is there and it looks like it would break functionality instead of fixing it.

Thanks for pointing that out.